### PR TITLE
Avoid slow query logging

### PR DESCRIPTION
**Pull request by SuperQ**
Add a session log_slow_filter[0] to avoid slow query logging of scrapes.
* Add a flag, default false, because this is not supported by Oracle MySQL.

[0]: https://www.percona.com/doc/percona-server/5.6/diagnostics/slow_extended.html

### CODE REVIEW

File: mysqld_exporter.go [hunk 1]
```diff
@@ -26,6 +26,10 @@ var (
 		"web.telemetry-path", "/metrics",
 		"Path under which to expose metrics.",
 	)
+	slowLogFilter = flag.Bool(
+		"log_slow_filter", false,
+		"Add a log_slow_filter to avoid exessive MySQL slow logging",
```
Comment (beorn7) [inline mysqld_exporter.go]: End with period for consistency (same below, actually…).

File: mysqld_exporter.go [hunk 2]
```diff
@@ -737,6 +742,14 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 	}
 	defer db.Close()
 
+	if *slowLogFilter {
+		sessionSettingsRows, err := db.Query(sessionSettingsQuery)
+		if err != nil {
+			return err
+		}
+		defer sessionSettingsRows.Close()
```
Comment (beorn7) [inline mysqld_exporter.go]: Is there a reason to defer the Close call to the end of the function? sessionSettingsRows goes out of scope here, so you could close it right ahead.

File: mysqld_exporter.go (hunk 1 above)
Comment (beorn7) [inline mysqld_exporter.go]: Perhaps add a comment that Oracly MySQL doesn't support it?

**Comment by SuperQ** [comment]
@beorn7 Updated with your requested changes.

**Comment by beorn7** [comment]
:+1:

**Pull request merged**